README.spanish.md

```diff
@@ -992,19 +1041,19 @@ Todo lo de encima hubiera devuelto `false` si se hubiera usado `===`
 
 **De lo contrario:** Si un atacante descubre que no está validando entradas externas proporcionadas por el usuario, puede aprovechar esta vulnerabilidad al publicar enlaces especialmente diseñados en foros, redes sociales y otros lugares públicos para que los usuarios hagan clic en él.
 
-🔗 [**Leer más: Prevent unsafe redirects**](/sections/security/saferedirects.spanish.md)
+🔗 [**Leer más: Evita redireccionamientos inseguros**](./sections/security/saferedirects.spanish.md)
 
 <br/><br/>
 
 ## ![✔] 6.25. Evita publicar secretos en el registro npm
 
 <a href="https://www.owasp.org/index.php/Top_10-2017_A6-Security_Misconfiguration" target="_blank"><img src="https://img.shields.io/badge/%E2%9C%94%20OWASP%20Threats%20-%20A6:Security%20Misconfiguration%20-green.svg" alt=""/></a>
 
-**TL;DR:** Se deben tomar precauciones para evitar el riesgo de publicar accidentalmente secretos en registros públicos de npm. Se puede usar un archivo `.npmignore` para poner en una lista negra archivos o carpetas específicos, o la matriz ` files` en `package.json` puede actuar como una lista blanca.
+**TL;DR:** Se deben tomar precauciones para evitar el riesgo de publicar accidentalmente secretos en registros públicos de npm. Se puede usar un archivo `.npmignore` para poner en una lista negra archivos o carpetas específicos, o el arreglo `files` en `package.json` puede actuar como una lista blanca.
```

Comment:
matriz is more like matrix. But mathematically speaking we do call a 2 dimensional array matriz.
Regardless of the dimensions, in programming the proper term si arreglo